src/components/Camera/index.js

```diff
@@ -25,14 +25,24 @@ export type Props = {
   onFailure?: Error => void,
   onUserMedia?: Function,
   webcamRef: React.Ref<typeof Webcam>,
-  video?: boolean
+  video?: boolean,
+  facing?: 'user' | 'environment',
```

Comment:
is environment for back capture?

Reply:
yeap, that's the naming convention used by the API. 
More details here if you're interested:
https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints/facingMode